buildSrc/src/main/java/org/elasticsearch/gradle/testclusters/ElasticsearchNode.java

```diff
@@ -575,6 +566,7 @@ private void createConfiguration()  {
             // We create hard links  for the distribution, so we need to remove the config file before writing it
             // to prevent the changes to reflect across all copies.
             Files.delete(configFile);
+            Files.write(configFile.getParent().resolve("unicast_hosts.txt"), "".getBytes(StandardCharsets.UTF_8));
```

Comment:
Do we need to provide an empty file for single node clusters?

Reply:
I think the file has to exist when the cluster starts, but will do another test to be sure.

Reply:
Turns out that's not the case, removed it, good find.